app/_src/kubernetes-ingress-controller/guides/redis-rate-limiting.md

```diff
@@ -279,7 +249,7 @@ kubectl patch kongplugin rate-limit --type json --patch '[
 
 ### Check plugin configuration
 
-The updated KongPlugin will propagate the reference to the proxy configuration.
+The updated KongPlugin propagates the reference to the proxy configuration.
 You can confirm it by checking the admin API.
 
 In one terminal, open a port-forward to the admin API:
```

Comment:
These instructions don't work. The deployment is called `kong-gateway` not `ingress-kong`

It's also missing a `-n kong`

Reply:
I can't add a comment to the line as it hasn't changed.

```
"{vault://env/secret-redis-password}"
Forwarding from 127.0.0.1:8444 -> 8444
```

Should be:

```
Forwarding from 127.0.0.1:8444 -> 8444
```